README.MD

```diff
@@ -119,6 +119,7 @@ render(
 | gridType            | oneOf(['grid', 'tree']) | whether the grid will be a flat list or a tree view |
 | data                | arrayOf(object)         | local data for grid to display, [more information available](https://github.com/bencripps/react-redux-grid/blob/master/docs/PROVIDING_DATA.md)|
 | dataSource          | func                    | function which returns data to display, [more information available](https://github.com/bencripps/react-redux-grid/blob/master/docs/PROVIDING_DATA.md)|
+| rowRenderer         | func                    | function which returns the row contents for this row, [more information available](https://github.com/bencripps/react-redux-grid/blob/master/docs/USING_CUSTOM_ROW_RENDERERS.md)|
```

Comment:
I think `rowRenderer` is a great idea, but I think the implementation should work through the`plugins` schema, since I can see future customization may we be desired for `Row`. Something like this: 

```js
const plugins = {
	ROW: {
		renderer: ({ }) => // return custom row component
	}
};
```

That way, it will be much simpler to extend this implementation. It's also less work, since you don't have to plumb the new props from grid => row.